corehq/motech/fhir/models.py

```diff
@@ -71,6 +72,19 @@
 from .validators import validate_supported_type
 
 
+class SQLFHIRRepeater(SQLCaseRepeater):
```

Comment:
This important takeaway here is that SQLFHIRRepeater is a subsubclass of SQLRepeater